Invalid client from Technic should not fail

```
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] net.technicpack.rest.RestfulAPIException: Error accessing URL [http://minecraft.webhippie.de/api/modpack/webhippie-tech/?cid=aaaaaaaa-da46-4766-a816-1b665c96cd7e]
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.rest.RestObject.getRestObject(RestObject.java:157)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.solder.http.HttpSolderPackApi.getPackInfo(HttpSolderPackApi.java:61)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.solder.http.HttpSolderPackApi.getPackInfoForBulk(HttpSolderPackApi.java:55)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.solder.cache.CachedSolderPackApi.pullAndCache(CachedSolderPackApi.java:112)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.solder.cache.CachedSolderPackApi.getPackInfoForBulk(CachedSolderPackApi.java:88)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.platform.PlatformPackInfoRepository.getInfoFromPlatformInfo(PlatformPackInfoRepository.java:73)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.platform.PlatformPackInfoRepository.getPlatformPackInfo(PlatformPackInfoRepository.java:60)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.platform.PlatformPackInfoRepository.getCompletePackInfo(PlatformPackInfoRepository.java:51)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.launchercore.modpacks.PackLoadJob$6.run(PackLoadJob.java:190)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at java.lang.Thread.run(Thread.java:745)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] Caused by: java.io.FileNotFoundException: http://minecraft.webhippie.de/api/modpack/webhippie-tech?cid=aaaaaaaa-da46-4766-a816-1b665c96cd7e
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1834)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1439)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	at net.technicpack.rest.RestObject.getRestObject(RestObject.java:137)
[B#347] 2016/10/02 18:06:42 [SCHWERWIEGEND] 	... 9 more
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/38048041-invalid-client-from-technic-should-not-fail?utm_campaign=plugin&utm_content=tracker%2F32209314&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32209314&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/router/middleware/session/solder.go
+++ b/router/middleware/session/solder.go
@@ -1,8 +1,6 @@
 package session
 
 import (
-	"net/http"
-
 	"github.com/gin-gonic/gin"
 	"github.com/kleister/kleister-api/store"
 )
@@ -16,19 +14,6 @@ func SetSolder() gin.HandlerFunc {
 				c.Query("k"),
 			)
 
-			if res.Error != nil {
-				c.JSON(
-					http.StatusNotFound,
-					gin.H{
-						"status":  http.StatusNotFound,
-						"message": "Failed to find key",
-					},
-				)
-
-				c.Abort()
-				return
-			}
-
 			if !res.RecordNotFound() {
 				c.Set(KeyContextKey, record)
 			}
@@ -39,19 +24,6 @@ func SetSolder() gin.HandlerFunc {
 				c,
 				c.Query("cid"),
 			)
-
-			if res.Error != nil {
-				c.JSON(
-					http.StatusNotFound,
-					gin.H{
-						"status":  http.StatusNotFound,
-						"message": "Failed to find client",
-					},
-				)
-
-				c.Abort()
-				return
-			}
 
 			if !res.RecordNotFound() {
 				c.Set(ClientContextKey, record)

